File: app/src/main/java/com/suhang/networkmvp/dagger/module/AttentionModule.java
package com.suhang.networkmvp.dagger.module;


import com.suhang.networkmvp.annotation.FragmentScope;
import com.suhang.networkmvp.mvp.base.IBlankView;

import dagger.Module;

/**
 * Created by 苏杭 on 2017/1/20 17:03.
 */

@FragmentScope
@Module
public class AttentionModule extends ParentModule<IBlankView>{
    public AttentionModule(IBlankView view) {
        super(view);
    }
}

File: app/src/main/java/com/suhang/networkmvp/dagger/module/AttentionModule.kt
package com.suhang.networkmvp.dagger.module

import com.suhang.networkmvp.mvp.model.AttentionModel
import com.suhang.networkmvp.mvp.model.IAttentionModel
import dagger.Module
import dagger.Provides

/**
 * Created by 苏杭 on 2017/6/1 20:44.
 */
@Module
class AttentionModule{
    @Provides
    fun prividerModel(): IAttentionModel {
        return AttentionModel()
    }
}
